contracts/0.4.24/Lido.sol

```diff
@@ -651,6 +673,156 @@ contract Lido is ILido, StETH, AragonApp {
         return EL_REWARDS_VAULT_POSITION.getStorageAddress();
     }
 
+    /**
+     * @dev updates beacon state and calculate rewards base (OUTDATED)
+     */
+    function _processAccounting(
+        // CL values
+        uint256 _beaconValidators,
+        uint256 _beaconBalance
+    ) internal returns (uint256 appearedValidators) {
+        uint256 depositedValidators = DEPOSITED_VALIDATORS_POSITION.getStorageUint256();
+        require(_beaconValidators <= depositedValidators, "REPORTED_MORE_DEPOSITED");
+
+        uint256 beaconValidators = BEACON_VALIDATORS_POSITION.getStorageUint256();
+        require(_beaconValidators >= beaconValidators, "REPORTED_LESS_VALIDATORS");
+
+        // Save the current beacon balance and validators to
+        // calculate rewards on the next push
+
+        BEACON_BALANCE_POSITION.setStorageUint256(_beaconBalance);
+
+        if (_beaconValidators > beaconValidators) {
+            BEACON_VALIDATORS_POSITION.setStorageUint256(_beaconValidators);
+        }
+
+        return _beaconValidators.sub(beaconValidators);
+    }
+
+    /**
+     * @dev move funds between ELRewardsVault, deposit and withdrawal buffers. Updates buffered counters respectively
+     */
+    function _processFundsMoving(
+        uint256[] _requestIdToFinalizeUpTo,
+        uint256[] _finalizationPooledEtherAmount,
+        uint256[] _finalizationSharesAmount,
+        uint256 _wcBufferedEther
+    ) internal returns (uint256) {
+        uint256 executionLayerRewards = 0;
+        address elRewardsVaultAddress = getELRewardsVault();
+        // If LidoExecutionLayerRewardsVault address is not set just do as if there were no execution layer rewards at all
+        // Otherwise withdraw all rewards and put them to the buffer
+        if (elRewardsVaultAddress != address(0)) {
+            executionLayerRewards = ILidoExecutionLayerRewardsVault(elRewardsVaultAddress).withdrawRewards(
+                (_getTotalPooledEther() * EL_REWARDS_WITHDRAWAL_LIMIT_POSITION.getStorageUint256()) / TOTAL_BASIS_POINTS
+            );
+        }
+
+        // Moving funds between withdrawal buffer and deposit buffer
+        // depending on withdrawal queue status
+        int256 movedToWithdrawalBuffer = _processWithdrawals(
```

Comment:
withdrawalsQueueBalanceChange

Reply:
Can be a bit misleading. Change relative to what? Previous report come to my mind at the first place.